scripts/cluster/agent.py

```diff
@@ -502,15 +501,15 @@ def update_dqlite_ip(host):
     :param : the host others see for this node
     """
     dqlite_port = get_dqlite_port()
-    subprocess.check_call("snapctl stop microk8s.daemon-apiserver".split())
+    service('stop', 'apiserver')
     time.sleep(10)
 
     cluster_dir = "{}/var/kubernetes/backend".format(snapdata_path)
     # TODO make the port configurable
     update_data = {'Address': "{}:{}".format(host, dqlite_port)}
     with open("{}/update.yaml".format(cluster_dir), 'w') as f:
         yaml.dump(update_data, f)
-    subprocess.check_call("snapctl start microk8s.daemon-apiserver".split())
+    service('start', 'apiserver')
```

Comment:
I think this will fail the black formatter?

Reply:
I think it is passing on these GH actions, right?